### DISCUSSION TITLE
Export a template user with no persistent modifications

### DISCUSSION THREAD

**Discussion by carlnx** [ltsp/ltsp#259] [Q&A]
Hi people, everything fine?

Please, I would like a little help. Here, where I work, I need to set up several public accesses where I only need to provide a desktop with FireFox, Chromium and Libreoffice (which I made various configs).

I set up an LTSP on Ubuntu Mate 18.04 and everything worked beautifully, even in chrootless mode.

I was also able to set up automatic login, using a more restricted user profile. The problem is that LTSP exports the user and allows permanent modifications. What I would really like is that every time I load a desktop, open the profile already configured and the modifications **are not persistent** and do not reflect in my local (server) profile.

Any idea?

Thanks,
Carlos Frederico.

**Comment by alkisg** [comment]
I think you're looking for something like this:
https://github.com/ltsp/community/wiki/Guest%20session
